Neos.ContentRepository.Core/Classes/Feature/ContentStreamHandling.php

```diff
@@ -133,25 +91,19 @@ private function forkContentStream(
      * @param ContentStreamId $contentStreamId The id of the content stream to remove
      * @phpstan-pure this method is pure, to persist the events they must be handled outside
      */
-    private function removeContentStream(
+    private function removeContentStreamWithoutConstraints(
```

Comment:
Same here: I would only add the suffix if there's a version _with_ constraint checks